src/middlewares/accomodation.middleware.js

```diff
@@ -76,5 +106,25 @@ class AccomodationMiddleware {
     req.body.checkoutDate = checkoutDate;
     next();
   }
+
+  /** This function if the the accommodation id provided is in our database
+   *  @param {req} req it contains the request from the body
+   *  @param {res} res it contains the response to be returned
+   *  @param {function} next it jumps to the next middleware in the route
+   * @return {object} the data from the first middleware
+   */
+  static async checkIfAccommodationIdExist(req, res, next) {
+    const { accomodationId } = req.params;
+    const accomodations = await accommodationService.checkAccommodationById(accomodationId);
+    if (!accomodations[0]) { return response.errorMessage(res, 'This accommodation does not exist', 401); }
+    const accommodationsRated = await accommodationService.getRatedAccommodations(accomodationId);
```

Comment:
Kindly stick to the scope of the function. This function does check if a particular accommodation exists and returns true or false!.

Reply:
I changed this and it's only checking.